catalogue.py

```diff
@@ -69,6 +69,101 @@ def reset(self):
+    FactionDetail(Faction.IMPERIALS, """The concept of raw power and strength has long been a core tenet of the
+                  self-dubbed Empire, with compulsory military service a cultural feature. Drilled into the populace
+                  for such an extensive period, the armed forces of the Imperials are a fearsome sight to behold.
+                  Those opposite gaze at one another, gauging whether it might be preferred to retreat. But this
+                  superiority leads to carelessness, as the Imperials assume that no one would dare attack one of
+                  their settlements for fear of retribution, and thus leave them relatively undefended.""",
+                  "+ Units have 50% more power", "- Settlements have 50% strength", VictoryType.ELIMINATION),
+    FactionDetail(Faction.PERSISTENT, """Atop a mountain in the north of these lands, there is a people of a certain
+                  philosophical nature. Instilled in all from birth to death is the ideal of determination, and
+                  achieving one's goals no matter the cost, in time or in life. Aptly dubbed by others as The
+                  Persistent, these militaristic people often elect to wear others down through sieges and defensive
+                  manoeuvres. Of course, such strategies become ineffective against the well-prepared, but this does
+                  not bother The Persistent; they simply continue on.""", "+ Units have 50% more health",
+                  "- Units have 75% of usual power", VictoryType.ELIMINATION),
+    FactionDetail(Faction.EXPLORERS, """Originating from an isolated part of the globe, the Explorers were first
+                  introduced to the wider world when a lost trader stumbled across their crude and underdeveloped
+                  settlement. Guiding the leaders of the settlement out to the nearest other settlement, and returning
+                  to explain to the masses was significant. Once the Explorers got a taste, they have not been able to
+                  stop. They look higher, run farther and dig deeper, at the expense of their energy levels.
+                  Unfortunately for the Explorers, the required rest during the journey makes them easy targets for
+                  Heathens.""", "+ Units have 50% more stamina", "- Units have 75% of usual health",
+                  VictoryType.GLUTTONY),
+    FactionDetail(Faction.INFIDELS, """Some say they were raised by Heathens, and some say that their DNA is actually
+                  closer to Heathen than human. Regardless of their biological makeup, if you approach someone on the
+                  street of any settlement and bring up the Infidels, you will be met with a look of disgust and the
+                  question 'you're not one of them, are you?'. Seen as sub-human, other empires engage in combat on
+                  sight with the Infidels, no matter the disguises they apply.""", "+ Not attacked by heathens",
```

Comment:
Just realised I missed something in your last refactor, should be easy enough to address in this one - `Not attacked by heathens` should be replaced by `Special affinity with heathens`.